python_code/3d/lr_set_matrix.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from VORTEXm import VORTEXm
 
-def lr_set_matrix(iwing, Xt, nXt, XC, NC):
+def lr_set_matrix2(iwing, Xt, nXt, XC, NC):
```

Comment:
```suggestion
def lr_set_matrix(iwing, Xt, nXt, XC, NC):
```
remove 2 from func name